data/json/items/containers.json

```diff
@@ -970,7 +968,7 @@
         "rigid": true,
         "watertight": true,
         "open_container": true,
-        "sealed_data": { "spoil_multiplier": 1.1 },
+        "sealed_data": { "spoil_multiplier": 1.0 },
```

Comment:
```suggestion
        "//": "sealed_data is here so the container can be sealed to avoid spilling",
        "sealed_data": { "spoil_multiplier": 1.0 },
```